// api/mockNews.js
export default [
    {
        id: "1",
        categoryId: "1",
        name: "News 1",
        description: " Каждый бот представляет собой набор индикаторов и параметров ,  Которые автоматически проводят сделку на бирже (или полуавтоматически: с подтверждением оператора)",
        price: 823,
        image: "/img/bot.jpg",
        link:'/news/1',
    },
    {
        id: "2",
        categoryId: "2",
        name: "News 2",
        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
        price: 953,
        image: "/img/bot2.jpg",
        link: '/news/2',

    },
    {
        id: "3",
        categoryId: "3",
        name: "News 3",
        description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam ultricies lorem odio, at laoreet tellus sodales in. Nullam maximus eros ut tortor ultricies rutrum. Aliquam euismod lacus non est egesta",
        price: 953,
        image: "/img/index-back-3.jpg",
        link: '/news/3',
    },]
